handbook/support.md

```diff
@@ -0,0 +1,13 @@
+# Requesting Support
+
+## Community Forum
+If you have any question like how to do ____ i.e. how to setup SFTP in EasyEngine, you should post in [community forum](https://community.easyengine.io/). There are helpful folks around who might be able to guide you 🕵🏻‍♂️️.
+
+## Slack
+For sharing feedback on EasyEngine and for discussion related to EasyEngine development, you can come by and say hello to us on slack :wave: [![Join EasyEngine Slack Channel](http://slack.easyengine.io/badge.svg)](http://slack.easyengine.io/)
+
+## Github
+If you think you've found a bug in EasyEngine, you can create the issue in [one of our github repositories](github-repos.md).
+
+## Paid Support
+ EasyEngine does not provide paid support as most of the time of core developers goes in building EasyEngine :rocket:. Hence they have very litte to no time left for serving support queries.
```

Comment:
EasyEngine's parent company rtCamp provides fully managed hosting & support retainers for ongoing needs. [You can contact them here](https://rtcamp.com/contact/). 

We have a directory of a select few EasyEngine experts who may be able to provide you premium support. You can get in touch with them through [EasyExperts directory](https://easyengine.io/easyexperts/).